src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs

```diff
@@ -49,7 +49,7 @@ internal static class RoslynDiagnosticIds
         public const string DoNotCallGetTestAccessorRuleId = "RS0043";
         public const string CreateTestAccessorRuleId = "RS0044";
         public const string ExposeMemberForTestingRuleId = "RS0045";
-
+        public const string AvoidOptSuffixForNullableEnableCodeRuleId = "RS0046";
```

Comment:
This was intentionally separated by a blank line for a new ID range used for temporary `RS01xx` diagnostics.

```suggestion
        public const string AvoidOptSuffixForNullableEnableCodeRuleId = "RS0046";
        
```